airflow/providers/google/cloud/hooks/cloud_build.py

```diff
@@ -207,6 +209,11 @@ def create_build(
         :param metadata: Optional, additional metadata that is provided to the method.
 
         """
+        warnings.warn(
```

Comment:
since we have merged https://github.com/apache/airflow/pull/30975 so we should use `AirflowProviderDeprecationWarning`.

Reply:
Thank you I fixed it.